web-apps/genesys-spark-examples/package.json

```diff
@@ -7,11 +7,12 @@
     "build": "webpack --mode=production",
     "postbuild": "npm run generate-versions-file",
     "clean": "rm -r ./dist || true",
-    "dev": "webpack serve --mode=development",
+    "dev": "npm run pre-dev && webpack serve --mode=development",
     "eslint": "eslint . --ignore-path .gitignore --fix",
     "generate-versions-file": "./scripts/generate-versions-file.mjs",
     "lint-all": "npm-run-all \"stylelint\" \"eslint\" \"prettier\" \"prettier-package-json\"",
     "lint-staged": "lint-staged",
+    "pre-dev": "./scripts/dev-examples-start.sh",
```

Comment:
FYI: https://docs.npmjs.com/cli/v10/using-npm/scripts#pre--post-scripts

tis could be `predev` and `dev` would not have to change at all